content/en/docs/tutorials/configuration/configure-redis-using-configmap.md

```diff
@@ -38,82 +36,218 @@ This page provides a real world example of how to configure Redis using a Config
 
 ## Real World Example: Configuring Redis using a ConfigMap
 
-You can follow the steps below to configure a Redis cache using data stored in a ConfigMap.
+Follow the steps below to configure a Redis cache using data stored in a ConfigMap.
 
-First create a `kustomization.yaml` containing a ConfigMap from the `redis-config` file:
-
-{{< codenew file="pods/config/redis-config" >}}
+First create a ConfigMap with an empty configuration block:
 
 ```shell
-curl -OL https://k8s.io/examples/pods/config/redis-config
-
-cat <<EOF >./kustomization.yaml
-configMapGenerator:
-- name: example-redis-config
-  files:
-  - redis-config
+cat <<EOF >./example-redis-config.yaml
+apiVersion: v1
```

Comment:
Hi @jai . Thanks for creating this pull request.
I ran the updated tutorial. The commands ran successfully except for line 57,
`kubectl apply -f redis-config.yaml`. Based upon line 44, `cat <<EOF >./example-redis-config.yaml`,
I would expect `kubectl apply -f example-redis-config.yaml`?

Reply:
Hi @jai.
Just checking if you have time to address the comments or need more information?

Reply:
On it as we speak 🚀

Reply:
fixed